### PR TITLE
small reformulation

### PR DESCRIPTION
**Pull request by RapTho**

### CODE REVIEW

File: 5-network/06-fetch-api/article.md [hunk 1]
```diff
@@ -152,13 +152,13 @@ The `credentials` option specifies whether `fetch` should send cookies and HTTP-
...
 
-By default, `fetch` requests make use of standard HTTP-caching. That is, it honors `Expires`, `Cache-Control` headers, sends `If-Modified-Since`, and so on. Just like regular HTTP-requests do.
+By default, `fetch` requests make use of standard HTTP-caching. That is, it respects the `Expires` and `Cache-Control` headers, sends `If-Modified-Since` and so on. Just like regular HTTP-requests do.
 
 The `cache` options allows to ignore HTTP-cache or fine-tune its usage:
 
 - **`"default"`** -- `fetch` uses standard HTTP-cache rules and headers,
 - **`"no-store"`** -- totally ignore HTTP-cache, this mode becomes the default if we set a header `If-Modified-Since`, `If-None-Match`, `If-Unmodified-Since`, `If-Match`, or `If-Range`,
-- **`"reload"`** -- don't take the result from HTTP-cache (if any), but populate cache with the response (if response headers allow),
+- **`"reload"`** -- don't take the result from HTTP-cache (if any), but populate the cache with the response (if response headers are allowed),
```
Comment (iliakan) [inline 5-network/06-fetch-api/article.md:161]: I meant: "if response headers allow that" =)

File: 5-network/06-fetch-api/article.md [hunk 2]
```diff
@@ -213,12 +213,12 @@ window.onunload = function() {
...
 
-Normally, when a document is unloaded, all associated network requests are aborted. But `keepalive` option tells the browser to perform the request in background, even after it leaves the page. So this option is essential for our request to succeed.
+Normally, when a document is unloaded, all associated network requests are aborted. But the `keepalive` option tells the browser to perform the request in the background, even after it leaves the page. So this option is essential for our request to succeed.
 
 It has a few limitations:
 
 - We can't send megabytes: the body limit for `keepalive` requests is 64KB.
     - If we need to gather a lot of statistics about the visit, we should send it out regularly in packets, so that there won't be a lot left for the last `onunload` request.
-    - This limit applies to all `keepalive` requests together. In other words, we can perform multiple `keepalive` requests in parallel, but the sum of their body lengths should not exceed 64KB.
+    - This limit applies to all `keepalive` requests. In other words, we can perform multiple `keepalive` requests in parallel, but the sum of their body lengths should not exceed 64KB.
```
Comment (iliakan) [inline 5-network/06-fetch-api/article.md:222]: I'd like to emphasize that the limit applies to all of them - not to any of them, but all together. What's wrong here? Should we rephrase it?

**Review by iliakan — CHANGES_REQUESTED** [review]
Please see the comments.

File: 5-network/06-fetch-api/article.md (hunk 1 above)
Comment (RapTho) [reply 5-network/06-fetch-api/article.md:161]: > I meant: "if response headers allow that" =)

How about "if the response headers permit this action" ? :)

File: 5-network/06-fetch-api/article.md (hunk 2 above)
Comment (RapTho) [reply 5-network/06-fetch-api/article.md:222]: > I'd like to emphasize that the limit applies to all of them - not to any of them, but all together. What's wrong here? Should we rephrase it?

To my subjective understanding the "together" could be omitted. You already say "all keepalive requests". Up to you :)

Comment (iliakan) [reply 5-network/06-fetch-api/article.md:222]: Please leave it be, for someone less smart than you are =)

File: 5-network/06-fetch-api/article.md (hunk 1 above)
Comment (iliakan) [reply 5-network/06-fetch-api/article.md:161]: A bit loo long, but that would be correct.

Comment (RapTho) [reply 5-network/06-fetch-api/article.md]: Edited it. Decline if you prefer the old way

File: 5-network/06-fetch-api/article.md (hunk 2 above)
Comment (RapTho) [reply 5-network/06-fetch-api/article.md]: Restored the "together" :)

**Comment by iliakan** [comment]
Thank you! =)

**Pull request merged**